internal/generator/fluentd/sources.go

```diff
@@ -100,9 +105,13 @@ func ContainerLogPaths() string {
 	return fmt.Sprintf("%q", "/var/log/pods/*/*/*.log")
 }
 
-func ExcludeContainerPaths(namespace string) string {
-	paths := []string{}
-	for _, comp := range []string{constants.CollectorName, constants.ElasticsearchName, constants.KibanaName} {
+func ExcludeContainerPaths(namespace string, outputs []string) string {
+	var paths []string
+	toExclude := []string{constants.CollectorName, constants.ElasticsearchName, constants.KibanaName}
+	if slices.Contains(outputs, constants.LokiName) {
+		toExclude = append(toExclude, constants.LokiName)
+	}
+	for _, comp := range toExclude {
 		paths = append(paths, fmt.Sprintf("\"/var/log/pods/%s_%s-*/*/*.log\"", namespace, comp))
 	}
```

Comment:
@jcantrill you mean something like this 
```suggestion
	paths = append(paths, fmt.Sprintf("\"/var/log/pods/%s_*/*/*.log\"", namespace))
```
exclude all logs from >` /var/log/pods/openshift-logging_*/*/*.log`?

Reply:
yes, but we need to ensure we can still collect for the eventrouter.  It may be we simply have to add a dedicated in_tail config for the event router unless we can say "exclude all of openshift-logging except the eventrouter"